scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -44,8 +44,8 @@ workflow GvsExtractCallset {
     Int? split_intervals_mem_override
     Float x_bed_weight_scaling = 4
     Float y_bed_weight_scaling = 4
-    Boolean write_cost_to_db = true
     Boolean is_wgs = true
+    Boolean convert_filtered_genotypes_to_nocalls = false
```

Comment:
Although not required by the corresponding Jira, I made this functionality available via the wdl - largely to allow me to test, but it seems like a possibly useful feature.

Reply:
smart